tests/resources/macros.hy

```diff
@@ -0,0 +1,4 @@
+(defmacro threadtail-set-ab []
+ (defn f [&rest args] (.join "" (+ (, "a") args)))
+  (setv variable (HySymbol (->> "b" (f))))
+  `(setv ~variable 2))
```

Comment:
Is this idiomatic style with the `join` on the same line as `defn`?

Reply:
Eh, I dunno. I forgot a space at the beginning of this line, though, so I've fixed that.